docker/Dockerfile_MSSQL

```diff
@@ -0,0 +1,50 @@
+FROM mcr.microsoft.com/mssql/rhel/server:2019-CU1-rhel-7.6
+
+# Change timezone to PST for convenience
+ENV TZ=PST8PDT
```

Comment:
I just realized we are using PST in all of our backup containers, wondering if it would be better/possible to use UTC instead and set the desired timezone as an environment variable when deploying the container instead.

This is NOT something we should look into during this PR, but I thought I'd point it out since I noticed it now.

Reply:
It's been set to PST to make it easy to schedule the cron jobs.